src/Exception/UnprocessableEntityHttpException.php

```diff
@@ -11,4 +11,15 @@
  */
 class UnprocessableEntityHttpException extends ClientErrorHttpException
 {
+    /**
+     * Returns the errors of the response if there are any
+     *
+     * @return array
+     */
+    public function getResponseErrors()
+    {
+        $decodedBody = json_decode($this->getResponseBody(), true);
```

Comment:
maybe check if $decodedBody is !== false (if, for an unknown reason, the response is not a json)